### PR TITLE
Popup-window

### PR DESCRIPTION
**Pull request by davidigbo**
In this pull request, I used a JavaScript array to store the detail of cards as objects and made a dynamic popup window of each card by using JavaScript.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -39,4 +36,175 @@ window.onresize = () => {
...
+
+  overlayModal.classList.remove('active');
+  navLinks.forEach((link) => {
+    if (w > 768) {
+      link.style.color = '#344563';
+    } else {
+      link.style.color = '#fff';
+    }
+  });
+}
```
Comment (thecodechaser) [inline script.js]: - [ ] You did a great job on the styles but for this project, you need to stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=z855fWEmkYWYXoJx-0) design as much as possible. Please make sure the project title, tags, and image start on the same vertical line and reduce the gap between the project title and tags.

### Your design

image

### Figma design

image

File: script.js [hunk 2]
```diff
@@ -39,4 +36,175 @@ window.onresize = () => {
...
+            <li>2015</li>
+          </ul>
+          </div>
+          <p class="content">${project.description}
+          </p>              
+          <ul class="skills project_${index}">
+          </ul>
+          <button class="button" data-project=${dataStr} data-toggle="modal" data-modal-target="#modal" > See project </button>
+         </div>
+        </div>`;
```
Comment (thecodechaser) [inline script.js:88]: - [ ] Please use the correct color for the model buttons' text. `#6070FF`

### Your design

image

### Figma design

image

File: script.js [hunk 3]
```diff
@@ -39,4 +36,175 @@ window.onresize = () => {
...
+  modal.classList.remove('active');
+  modal.innerHTML = '';
+
+  overlayModal.classList.remove('active');
+  navLinks.forEach((link) => {
+    if (w > 768) {
+      link.style.color = '#344563';
+    } else {
+      link.style.color = '#fff';
+    }
```
Comment (thecodechaser) [inline script.js]: - [ ] On mobile screens please make sure there's equal padding on left-right and content is in the center of the model to follow the Figma design.

### Your design

image

### Figma design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @david30-maker, and Team👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Nice UI design. ✔️
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (Abenezer-Tilahun) [reply script.js]: Fixed.

File: script.js (hunk 2 above)
Comment (Abenezer-Tilahun) [reply script.js:88]: Fixed.

File: script.js (hunk 3 above)
Comment (Abenezer-Tilahun) [reply script.js]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @david30-maker 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git Hub-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omar-muhamad — COMMENTED** [review]
# Invalid ❌

Hi @david30-maker,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Your Code Review Request is marked as invalid for these reasons:

- [ ] Please make a new branch with a new Pull Request for every project milestone you have.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

**Review by rloterh — COMMENTED** [review]
Hi David, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Your Pull Request title and description does not match with the activity under which you have submitted this project, and similarly, your project appears already approved under a different activity. Please ensure to create a new Pull Request for a new activity, and project should be submitted under the appropriate activity. 

Otherwise, kindly leave a comment in the PR why you intend resubmitting this approved project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**